system/include/libc/wchar.h

```diff
@@ -176,8 +172,11 @@ int       iswctype(wint_t, wctype_t);
 wint_t    towlower(wint_t);
 wint_t    towupper(wint_t);
 wctype_t  wctype(const char *);
+
+#ifndef __cplusplus
 #undef iswdigit
-#define iswdigit(a) ((unsigned)(a)-'0' < 10)
+#define iswdigit(a) (0 ? iswdigit(a) : ((unsigned)(a)-'0') < 10)
```

Comment:
Apparently it's about better compile-time diagnostics for misuse: https://github.com/bpowers/musl/commit/2ca55a93f2a11185d72dcb69006fd2c30b5c3144

The functions (including `iswdigit`) are declared, but only defined in terms of macros, and then the `(unsigned)` cast loses type information. So if someone calls them with an invalid argument type the frontend doesn't complain.